lib/Core/SymbolicError.h

```diff
@@ -178,6 +178,10 @@ class SymbolicError {
     return constraintsWithError;
   }
 
+  std::map<std::string, ref<Expr> > getErrorExpressions() {
```

Comment:
@Himeshi Here also, I would recommend:
```
std::map<std::string, ref<Expr> > &getErrorExpressions() {
```
instead (with ampersand) for efficiency.